azure/metric-alerts.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "metricAlertsArray": {
+      "type": "array"
```

Comment:
Can we add a description with an example of the object that goes into the array?